src/css/base.scss

```diff
@@ -110,11 +105,9 @@ main {
 }
 
 .instruction-content p {
-  padding: .5em;
+  padding: 0.5em;
```

Comment:
Is the '0' in front of your decimals necessary?  Or are you doing that for your own readability?

Reply:
Readability. It isn't necessary but is just what I've always done.